### ISSUE TITLE
STIL (Deepfake Detector) test on real videos

### ISSUE THREAD

**Issue by sohailahmedkhan** [Tencent/TFace#76]
Dear Authors,

thanks for making this awesome work on deepfake detection opensource. I am testing the provided STIL deepfake detection model on real videos from the FaceForensics++ dataset, however, the model seems to classify real videos as fake. I am also unable to find in the paper the detection performance on real samples.

Please guide.

**Comment by wizyoung** [comment]
@sohailahmedkhan How did you test STIL on FF++ dataset? Did you follow the guide to detect the faces and train the model?

**Comment by sohailahmedkhan** [comment]
I have extracted face cropped frames from REAL as well as FAKE videos. Then I loaded those images and converted to tensors, reshaped according to the desired input shape of STIL, and the fed to the model. 

I loaded the pretrained weights of the STIL model as well, which are provided in the code.

**Comment by wizyoung** [comment]
But we didn't provide the pretrained weights. Your mentioned "pretrained weights", I guess, is the ImageNet pretrained weights for the backbone.

**Comment by sohailahmedkhan** [comment]
import math
import torch
import torch.nn as nn
import torch.nn.functional as F
import torch.utils.model_zoo as model_zoo
import numpy as np


model_urls = {
    'scnet50_v1d': 'https://backseason.oss-cn-beijing.aliyuncs.com/scnet/scnet50_v1d-4109d1e1.pth',
}


class ISM_Module(nn.Module):
    def __init__(self, k_size=3):
        """The Information Supplement Module (ISM).

**Comment by sohailahmedkhan** [comment]
They are in the models/ops.py file

**Comment by wizyoung** [comment]
Yes, they are ImageNet pretrained weights for the backbone. And you need to train the model on FF++ dataset yourself.

**Comment by sohailahmedkhan** [comment]
Is it possible to get the pretrained weights?

**Comment by wizyoung** [comment]
We suggest following the guide and training the model yourself.

**Comment by sohailahmedkhan** [comment]
Unfortunately, I don't have the required resources to train the model myself. I would really appreciate if you can provide the pretrained weights. Thanks
